gulpfile.js

```diff
@@ -68,4 +69,9 @@ gulp.task('sass', function () {
         .pipe(gulp.dest('resources/css'));
 });
 
+gulp.task('install-hooks', function () {
```

Comment:
Add that this command needs to be executed before developing to the development section in the `README.md`.

Reply:
I have added the instructions to the installing section in the README.md now. If development is better, I will update the pull request.